src/document/generator.ml

```diff
@@ -703,7 +703,9 @@ module Make (Syntax : SYNTAX) = struct
         let markers = Syntax.Comment.markers in
         try
           let url = Url.Anchor.polymorphic_variant ~type_ident item in
-          let attrs = [ "def"; Url.Anchor.string_of_kind url.kind ] in
+          let attrs =
+            [ "def"; "poly-variant"; Url.Anchor.string_of_kind url.kind ]
+          in
```

Comment:
I'm not sure exactly why you had to change the generator to fix this issue (`odig`'s stylesheet have no problem with this without that) but, why not…

Reply:
Thinking more about this I'm not sure it's really a good idea to add more to the markup to distinguish polymorphic variants from regular ones. I don't really see a good use case for it.

Reply:
I think it is related with odoc doing something more complex, when the size of the comment is small, the comment is in the same line and takes 50% of the space, otherwise it takes more space and is wrapped below (see the screenshot or [some file where it happens](https://ocaml-ppx.github.io/ppxlib/ppxlib/Astlib/Ast_500/Parsetree/index.html)).
I don't know if that's worth the trouble, but I think that's the difference with `odig`, which displays comments in a simpler way (from a quick look at the stylesheet, I might be wrong).

There might be a way to do it without, but since variant and fields had their classes, it felt natural to add one for polymorphic variants